addons/binding/org.openhab.binding.elkm1/OSGI-INF/ElkM1HandlerFactory.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+	Copyright (c) 2010-2019 by the respective copyright holders.
+
+	All rights reserved. This program and the accompanying materials
+	are made available under the terms of the Eclipse Public License v1.0
+	which accompanies this distribution, and is available at
+	http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="binding.elkm1">
+
+   <implementation class="org.openhab.binding.elkm1.internal.ElkM1HandlerFactory"/>
```

Comment:
Please use DS annotations instead of adding these XML files:

* Remove this file 
* Add a `.gitignore` file to this directory ignoring all XML file i.e.: `*.xml`
* Annotate `ElkM1HandlerFactory` with:

```java
@Component(service = ThingHandlerFactory.class, configurationPid = "binding.elkm1")
```